data/heliarch intro.txt

```diff
@@ -0,0 +1,1568 @@
+			choice
+				`	(Dig out the bushes.)`
+					goto bushending
+				`	(Head back to the open warehouses and take shelter there.)`
+					goto beetledigger
+			label beetledigger
+			`	Though you can barely hear each other in the rain, Kirrie insists in staying there and digging through the ground for a while, to see if he finds anything.`
+			`	You walk back to the empty streets, taking care to not slip on the damp soil, and enter one of the warehouses Kirrie had previously opened. After a few minutes, he walks in, saying he dug out many, but couldn't find anything under the sets of bushes.`
+			`	Once the weather settles, near nightfall, he gets notified by Iplora that she and Tugroob are already back at the ship, so you two decide to head back as well.`
+			`	It seems Tugroob found some evidence of some smuggled weapons passing through here, but nothing major. They get back in their bunks, saying you may depart now.`
+				accept
+			label lonebeetle
+			`	He leaves too, and you're on your own in the ship for a couple hours. Eventually the get back, and it seems only Tugroob found something, mentioning some evidence of smuggled weapons passing through the planet.`
+			`	The three agents get back in their bunks, and say you may depart now.`
+
+
+
+mission "HI2-SF"
+	name "Investigate Stronghold of Flugbu"
+	description "Transport the Heliarch trio here so that they can investigate signs of Resistance activity."
+	to offer
+		has "Heliarch Investigation 1: done"
+	source "Ring of Friendship"
+	destination "Stronghold of Flugbu"
+	to fail
+		or
+			has "Heliarch Investigation 2: declined"
+			has "Heliarch Investigation 2: aborted"
+			has "Heliarch Investigation 2 :failed"
+	on complete
+		conversation
+			`While you doubt anyone opposing the Heliarchs would find much success in this massive compound, it seems the planet holds enough resources, ships and a valuable positioning that it was still marked for investigation.`
+			`	The three agents say you cannot go to most of the parts here they will be headed to, so you stay inside your ship. They return right as the sun is setting, and say they haven't found anything.`
+
+
+
+mission "HI2-SS"
+	name "Investigate Shifting Sand"
+	description "Transport the Heliarch trio here so that they can investigate signs of Resistance activity."
+	to offer
+		has "Heliarch Investigation 1: done"
+	source "Ring of Friendship"
+	destination "Shifting Sand"
+	to fail
+		or
+			has "Heliarch Investigation 2: declined"
+			has "Heliarch Investigation 2: aborted"
+			has "Heliarch Investigation 2 :failed"
+	on complete
+		conversation
+			`Immediately upon landing, Iplora is ready to leave your ship, going off on her own. You ask Tugroob and Kirrie about it, and they say that she is just taking a small break.`
```

Comment:
`Immediately upon landing, Iplora is the first to leave your ship, disappearing into the spaceport before you could stop her. You ask Tugroob and Kirrie about it, and they say that she is just taking a small break.`